### PR TITLE
New Project page redesign

### PR DESCRIPTION
**Pull request by marcosjr182**
- [x] Moves New Project page content to a panel.
- [x] Match projects form with other forms. 
- [x] Fix a positioning bug on the Integrations page.
- [x] Fix initial column sizes at the projects board

#### Tests causing build to fail [(fixed on (5c4a5fbe63))](https://github.com/Codeminer42/cm42-central/pull/110/commits/5c4a5fbe632d36d8317506eef689696c8db52d55)
There were a couple recurrent tests failing the builds. Apparently, adding the line `first('.column_header').click` was able to allow the page to load and the tests to pass more frequently. 
Increasing the `Capybara.default_max_wait_time` was not making any difference.

### Before
extrablankspace
old_newproject

### After
newproject

### CODE REVIEW

File: app/assets/stylesheets/_form.scss [hunk 1]
```diff
@@ -24,3 +24,14 @@
     }
   }
 }
+
+.date-form-group .auth-form-control {
+  width: auto;
```
Comment (marclerodrigues) [inline app/assets/stylesheets/_form.scss]: If you're not following any style guides, order the properties in alphabetical order.

File: app/assets/stylesheets/_form.scss [hunk 2]
```diff
@@ -24,3 +24,14 @@
...
+  display: inline;
+}
+
+.btn-form {
+  padding: 7px 20px;
```
Comment (marclerodrigues) [inline app/assets/stylesheets/_form.scss]: The same here

File: app/assets/stylesheets/_layout.scss [hunk 3]
```diff
@@ -24,7 +23,10 @@ body {
 }
 
 .content-wrapper {
-  height: 100%;
-  width: 100%;
+  top: 0;
```
Comment (marclerodrigues) [inline app/assets/stylesheets/_layout.scss]: And here

File: spec/support/integration_helpers.rb [hunk 4]
```diff
@@ -45,4 +45,8 @@ def wait_spinner
     expect(page).not_to have_css('.loading-spin.show')
   end
 
+  def wait_page_load
+    find('.column_header', match: :first).click
+  end
+
```
Comment (marclerodrigues) [inline spec/support/integration_helpers.rb]: Remove this line. :nail_care:

File: app/assets/stylesheets/_form.scss (hunk 1 above)
Comment (marcosjr182) [inline app/assets/stylesheets/_form.scss]: I got it, thanks!

**Comment by marclerodrigues** [comment]
LGTM, just check what happened with Travis. :+1:

**Pull request merged**